redhat-cop/tower_utilities
# Add RPM pre-requisites to pre_tasks and simplify role

### What does this PR do?

On a minimal RHEL 8, the unarchive task complains about
`Commands "gtar" and "tar" not found. Command "unzip" not found.`
Also there is no point in having a tasks file beside main, also removed
unneeded prefix to task names.

### How should this be tested?

Install Tower on RHEL8 with only the following packages and see how it complains without this PR:

```
%packages
@^minimal-environment
kexec-tools
%end
```

Then see that it works with the patch.

### Is there a relevant Issue open for this?

n/a

### Other Relevant info, PRs, etc.

n/a
